obsei/source/pandas_source.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from obsei import configuration
+from typing import List, Optional, Dict, Any
+
+from pandas.core.frame import DataFrame
+from obsei.source.base_source import BaseSource, BaseSourceConfig
+from obsei.payload import TextPayload
+from obsei.misc.utils import (
+    DATETIME_STRING_PATTERN,
+    DEFAULT_LOOKUP_PERIOD,
+    convert_utc_time,
+)
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+class PandasSourceConfig(BaseSourceConfig):
+    TYPE: str = "PandasScrapper"
```

Comment:
Correct the name it should be "Pandas" only

Reply:
Okay sir, I'll change it to pandas.